src/UI/Pages/Resources/ResourceTableRow.tsx

```diff
@@ -46,20 +46,15 @@ export const ResourceTableRow: React.FC<Props> = ({
         <Td dataLabel={words("resources.column.deployState")}>
           <ResourceStatusLabel status={row.deployState} />
         </Td>
-        <Td>
+        <StyledCell>
           <Link
-            to={{
-              pathname: routeManager.getUrl("ResourceDetails", {
-                resourceId: row.id,
-              }),
-              search: location.search,
-            }}
+            pathname={routeManager.getUrl("ResourceDetails", {
```

Comment:
I think there is a `ResourceLink` component that also leads to the `ResourceDetails` page, we could reuse that here